modules/creating-infra-machines.adoc

```diff
@@ -1,11 +1,12 @@
 // Module included in the following assemblies:
 //
+// * machine_management/creating-infrastructure-machinesets.adoc
```

Comment:
In line 30; this example shows 'worker' does that also need to be updated?

Reply:
@yuqi-zhang this originates from the [custom-pools.md](https://github.com/openshift/machine-config-operator/blob/master/docs/custom-pools.md) file managed by the MCO team. Specifically, [this line](https://github.com/openshift/machine-config-operator/blame/master/docs/custom-pools.md#L47). Should we remove the `worker` reference there? Or is that something required by the MCO?

Reply:
No, you leave it.  This is how the MCO applies the existing "worker" MachineConfigs to this new "infra" MCP.  You are basically stating that you want all MachineConfigs that are labeled either worker or infra (inclusive or) .

Reply:
@briantward thanks for confirming!

Reply:
I'll add a note here to clarify. I see where this could be confusing since the end goal is to create an MCP for infra nodes only.

Reply:
@briantward does this make sense as a note?:
> Custom machine config pools inherit machine configs from the worker pool. Custom pools use any machine config targeted for the worker pool, but add the ability to also deploy changes only targeted at the custom pool. Since a custom pool inherits resources from the worker pool, any change to the worker pool also affects the custom pool.

Reply:
"Sounds" good to me, but I'll let @kikisdeliveryservice confirm verbiage from MCO perspective.  I suppose you can get nitpicky and say "any change to a machineconfig labeled worker will also affect the custom pool".